tools/functions.sh

```diff
@@ -1,14 +1,15 @@
+#!/bin/sh
```

Comment:
This script is not meant to be executed, so [use a directive](https://github.com/koalaman/shellcheck/wiki/SC2148) `# shellcheck shell=sh`.

Reply:
Followed your advice